src/app/pages/community-page/community-page.component.html

```diff
@@ -65,10 +68,10 @@ <h2>Members {{ communityData.joinedAmount }}</h2>
 
 <ng-template #notAuthorizedButtons>
   <div class="buttons-container">
-    <button type="button" (click)="onNonAuthorizedClick()">
+    <button type="button" (click)="onNonAuthorized()">
       <mat-icon>add</mat-icon>
       Create a post
     </button>
-    <button type="button" (click)="onNonAuthorizedClick()">Join</button>
+    <button type="button" (click)="onNonAuthorized()">Join</button>
```

Comment:
Using double request of this function ( onNonAuthorized() ) for redirect creating post to authorization ?